crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/mock_call.rs

```diff
@@ -9,26 +11,30 @@ impl CheatnetState {
         &mut self,
         contract_address: ContractAddress,
         function_selector: EntryPointSelector,
+        call_data: Option<Vec<Felt>>,
         ret_data: &[Felt],
         span: CheatSpan,
     ) {
         let contract_mocked_functions = self.mocked_functions.entry(contract_address).or_default();
-
-        contract_mocked_functions.insert(
-            function_selector,
-            CheatStatus::Cheated(ret_data.to_vec(), span),
-        );
+        let call_data_hash = match call_data {
```

Comment:
Nitpick, but in most places in the code we already call it `calldata`, not `call_data`.

Reply:
Source code updated to use `calldata` instead of `call data`